sqlglot/dialects/hive.py

```diff
@@ -771,6 +851,14 @@ def alterset_sql(self, expression: exp.AlterSet) -> str:
 
             return f"SET{serde}{exprs}{location}{file_format}{tags}"
 
+        def alter_sql(self, expression: exp.Alter) -> str:
+            if self.dialect.CHANGE_COLUMN_STYLE == "HIVE":
+                actions = expression.args["actions"]
+                for action in actions:
+                    if isinstance(action, exp.RenameColumn):
+                        self.unsupported("Cannot rename columns without data type defined in Hive")
```

Comment:
So the idea here is that `RenameColumn` cannot be produced by Hive's parser at all? Can't we send a simple `find_all` to `expression` instead of manual for loops here?